### PR TITLE
Add serp events to levers daily

### PR DESCRIPTION
**Pull request by m-d-bowerman**
Adds the data from `serp_events` to the `search_revenue_levers_daily` table. The fields being added mostly correspond to what exists in `search_revenue_levers_daily` pulled from the Glean SERP telemetry. 

Kicks off a backfill of `search_revenue_levers_daily` to incorporate the full history of the SERP data.

## Related Tickets & Documents
* DENG-XXXX
* DSRE-XXXX

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/.github/reviewer_checklist.md)**

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-5012)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql [hunk 1]
```diff
@@ -275,6 +275,43 @@ combined_search_dau AS (
...
+    COUNTIF(is_tagged) AS serp_events_tagged_sap,
+    COUNTIF(is_tagged AND REGEXP_CONTAINS(sap_source, 'follow_on')) AS serp_events_tagged_follow_on,
+    SUM(num_ad_clicks) AS serp_events_ad_click,
+    COUNTIF(num_ads_visible > 0) AS serp_events_search_with_ads,
+    COUNTIF(NOT is_tagged) AS serp_events_organic,
+    SUM(IF(NOT is_tagged, num_ad_clicks, 0)) AS serp_events_ad_click_organic,
+    COUNTIF(num_ads_visible > 0 AND NOT is_tagged) AS serp_events_search_with_ads_organic,
+    -- serp_events does not have distribution ID or partner codes to calculate monetizable SAP
+    COUNTIF(ad_blocker_inferred) AS serp_events_sap_with_ad_blocker_inferred,
+    SUM(num_ads_visible) AS serp_events_ad_impression,
```
Comment (skahmann3) [inline sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql:299]:
```suggestion
    SUM(num_ads_visible) AS serp_impression_num_ads_visible,
```

File: sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql [hunk 2]
```diff
@@ -275,6 +275,43 @@ combined_search_dau AS (
...
+    SUM(num_ads_blocked) AS serp_events_ad_blocked,
```
Comment (skahmann3) [inline sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql:300]:
```suggestion
    SUM(num_ads_blocked) AS serp_impression_num_ad_blocked,
```

**Comment by alekhyamoz** [comment]
@m-d-bowerman If we add the columns to the existing table backfilling the new columns would impact the existing values (and will require validation effort)
If no backfilling is required, I do not see any harm in adding the columns to the table.

**Comment by m-d-bowerman** [comment]
> @m-d-bowerman If we add the columns to the existing table backfilling the new columns would impact the existing values (and will require validation effort) If no backfilling is required, I do not see any harm in adding the columns to the table.

The backfill is necessary, so that we have the full SERP event history included with the other search metrics in this table. I'll do the data validation for this backfill.

**Review by skahmann3 — APPROVED** [review]

**Pull request merged**